lab7/install/lab_7.rosinstall

```diff
@@ -7,9 +7,9 @@
     uri: https://github.com/ethz-asl/glog_catkin.git
     version: master
 - git:
-    local-name: gtsam_catkin
-    uri: https://github.com/ethz-asl/gtsam_catkin.git
-    version: master
+    local-name: gtsam
+    uri: git@github.com:borglab/gtsam.git
```

Comment:
```suggestion
    uri: https://github.com/borglab/gtsam.git
```

Using a raw url (instead of the better practice of using the `git@` form like you did) works for public repos even if you don't have ssh keys set up for `github.com`